cmd/cli/kubectl-kyverno/processor/result.go

```diff
@@ -75,11 +75,7 @@ func (rc *ResultCounts) addGenerateResponse(auditWarn bool, response engineapi.E
 				if ruleResponse.Status() == engineapi.RuleStatusPass {
 					rc.Pass++
 				} else {
-					if auditWarn && response.GetValidationFailureAction().Audit() {
-						rc.Warn++
-					} else {
-						rc.Fail++
-					}
+					rc.Fail++
```

Comment:
Can you elaborate on why we changed this?

Reply:
It is used for the generate rules in the CLI. 

Previously, when the flag '--auditWarn' was set and `validationFailureAction` was set to 'Audit', we considered the result as a warning. Otherwise, it was considered a failure. 

In a real cluster, `validationFailureAction` has no effect on the generate rules, which is why I removed the condition.